lib/twilio/uuid.ts

```diff
@@ -4,47 +4,42 @@
  * @internalapi
  */
 
-import * as md5Lib from 'md5';
 import { NotSupportedError } from '../twilio/errors';
 
-// If imported as an ESM module, sometimes md5 is available by accessing
-// the "default" property of the imported library.
-// @ts-ignore
-const md5 = typeof md5Lib === 'function' ? md5Lib : md5Lib.default;
-
-function generateUuid(): string {
+/**
+ * Generates a 16 character long random string where each character is from the
```

Comment:
Why 16? The string should be 32 characters excluding the prefix.

Reply:
MD5 hashes are 16 characters long. All our SIDs (including CallSids, CallMessageSids, etc.) are also 16 characters long no? https://app.circleci.com/pipelines/github/twilio/twilio-voice.js/18866/workflows/ef29c2d9-01f7-425d-a615-758767923bed/jobs/33286?invite=true#step-107-1184_43

Reply:
Doh, I had bytes and characters mixed up. Too used to interchanging the two from working in C-like languages. I will fix this, oopsies.